README.md

```diff
@@ -1 +1,68 @@
-# Todo_list_webpack
\ No newline at end of file
+# Todo_list_webpack
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Todo-list
+
```

Comment:
- [ ] great job with the project. kindly add a brief description about the project